packages/visualizations/src/components/Table/Pagination/Pagination.svelte

```diff
@@ -0,0 +1,55 @@
+<script lang="ts">
+    import Pages from './Pages.svelte';
+    import Numbering from './Numbering.svelte';
+    // import PerPage from "./PerPage.svelte";
+    import type { TableOptions } from '../types';
+
+    export let onChangePage: Required<TableOptions>['pages']['onChangePage'];
+    export let totalRecords: number;
+    export let recordsPerPage: number;
+    export let initial: number;
+
+    let current = initial;
+
+    const setPage = async (page: number) => {
+        onChangePage(page);
+        current = page;
+    };
+
+    // "floor" because it's ceil but starts at 0
+    const totalPages = Math.floor(totalRecords / recordsPerPage);
+</script>
+
+<div class="pagination">
+    <div class="numbering">
+        <Numbering
+            current={[recordsPerPage * (current - 1) + 1, recordsPerPage * current]}
```

Comment:
last page might not contain a number of records equal to recordsPerPage. 
10 records per page
15 records in total
so two pages
`recordsPerPage * current` gives for the last page => 20 (we need 15)